dev-python/internetarchive/internetarchive-3.7.0.ebuild

```diff
@@ -1,20 +1,27 @@
-# Copyright 2024 Gentoo Authors
+# Copyright 2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
-PYTHON_COMPAT=( python3_{11..12} )
+PYTHON_COMPAT=( python3_{11..13} )
 DISTUTILS_USE_PEP517=setuptools
 inherit distutils-r1
 
 DESCRIPTION="Lib and CLI for archive.org - for search, uploading, downloading, rename..."
 HOMEPAGE="https://github.com/jjjake/internetarchive"
-SRC_URI="https://github.com/jjjake/$PN/archive/refs/tags/v$PV.tar.gz"
+SRC_URI="https://github.com/jjjake/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
To make it even better it should be ${P}.gh.tar.gz as the source code can also be grabbed from https://pypi.org/project/internetarchive/